### ISSUE TITLE
Accept posting without timestamp

### ISSUE THREAD

**Issue by pgdr** [FriskByBergen/friskby#206]
If I submit the value `(value, sensor_id, timestamp)`  with `timestamp = null`, I would like to have the server interpret it as `now`.

Then it is up to the client to _not cache_ values, in a _fire-and-forget_ style.

**Comment by joakim-hove** [comment]
Maybe we could send a srring like 'now' or 'server'?

27. mars 2017 13:31 skrev "Pål Grønås Drange" <notifications@github.com>:

**Comment by pgdr** [comment]
Yes, we could.  Are you afraid that we might wrongly interpret (or transmit) a `null` or `0` as `timestamp(0)`?

It sounds easier on the server side to use 0, as we can then `if not timestamp: timestamp = now()`.

**Comment by joakim-hove** [comment]
My main point was that the client should send *something*, and not just an empty value. I might have misunderstood your original suggestion.
